mscore/inspector/inspectorTextLineBase.cpp

```diff
@@ -197,6 +202,76 @@ void InspectorTextLineBase::valueChanged(int idx)
             inspector->update();
       }
 
+//---------------------------------------------------------
+//   hasBeginTextClicked
+//---------------------------------------------------------
 
-} // namespace Ms
+void InspectorTextLineBase::hasBeginTextClicked(bool checked)
+      {
+      if (!checked) {
+            TextLineBaseSegment* ts = static_cast<TextLineBaseSegment*>(inspector->element());
+            TextLineBase* t = ts->textLineBase();
+
+            t->setBeginText(QString());
+            t->resetProperty(Pid::BEGIN_TEXT_PLACE);
+            t->resetProperty(Pid::BEGIN_TEXT_ALIGN);
+            t->resetProperty(Pid::BEGIN_FONT_FACE);
+            t->resetProperty(Pid::BEGIN_FONT_SIZE);
+            t->resetProperty(Pid::BEGIN_FONT_STYLE);
+            t->resetProperty(Pid::BEGIN_TEXT_OFFSET);
```

Comment:
These changes (like all the other editing actions) should be undoable, so they should be enclosed within `startCmd()`/`endCmd()` calls and use `undo*` versions of these functions.
Also it would be good to ensure that these checkboxes state is updated properly on undo/redo, but this might be not relevant for end user experience for now as undo/redo operations seem to not preserve selection on a text line.